### DISCUSSION TITLE
Lots of Chemistry improvements

### DISCUSSION THREAD

**Discussion by jlvaillant** [jlvaillant/intellicenter#20] [General]
Lots of improvements for those with IntelliChem and IntelliChlor systems (new switches, more sensors...).
Looks like the integration can handle a really complex setup now. Happy me :-)
Thanks to super contributor @dwradcliffe and new contributor @rsnodgrass for the changes and to all who took the time to help debug the issues and verify the fixes.

<hr /><em>This discussion was created from the release <a href='https://github.com/jlvaillant/intellicenter/releases/tag/1.1'>Lots of Chemistry improvements</a>.</em>

**Comment by abenedet78** [comment]
Absolutely love the latest improvements!   Thank you so much.  For me, there are only two missing functionalities left that I get in the Pentair app that I can't get with your integration.  1. Set pool heater temperature 2. Change the colour of the lights.  Is it possible to add these as well?

**Comment by jlvaillant** [comment]
Hmmmm, these 2 features should work. Pool temperature the the Water Heater object. Color should be under Effects for the lights (though I only tested with IntelliBrite). Do you know what kind of lights you have?
